common/src/main/java/no/difi/meldingsutveksling/config/IntegrasjonspunktProperties.java

```diff
@@ -90,14 +97,53 @@ public FeatureToggle getFeature() {
 
         private String username;
         private String password;
-        /**
-         * TODO: descrive
-         */
         private String externalServiceCode;
-        /**
-         * TODO: descrive
-         */
         private String externalServiceEditionCode;
+        private Sms sms;
+        @Valid
+        private Email email;
+
+        public class EmailConstraintValidator implements ConstraintValidator {
+
+            @Override
+            public void initialize(Annotation constraintAnnotation) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1186)